Unable to customise theme breakpoints

<!-- Provide a general summary of the issue in the Title above -->

<!--
  Thank you very much for contributing to Material-UI by creating an issue! ❤️
  To avoid duplicate issues we ask you to check off the following list.
-->

<!-- Checked checkbox should look like this: [x] -->

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

<!-- Describe what happens instead of the expected behavior. -->
No longer able to customise theme breakpoints.

## Expected Behavior 🤔

<!-- Describe what should happen. -->
Be able to import the `BreakpointOptions` type from core and use it to pass in custom breakpoints to theme.

## Steps to Reproduce 🕹

https://codesandbox.io/s/basicalerts-material-demo-forked-7qnte?file=/demo.tsx:9-27

```jsx
import { BreakpointsOptions } from "@material-ui/core/styles/createBreakpoints";

const getBreakpoints = (theme: CustomerUITheme): BreakpointsOptions => ({
    values: theme.breakpoints,
});
```

shows error `Cannot find module '@material-ui/core/styles/createBreakpoints`

there doesn't appear to be any error passing breakpoints into the `createTheme` function, and i believe `createBreakpoints` has moved to the system package.
Do we need to install `@material-ui/system` to be able to use the `BreakpointsOptions` type?

--- a/packages/material-ui-system/src/createTheme/createBreakpoints.d.ts
+++ b/packages/material-ui-system/src/createTheme/createBreakpoints.d.ts
@@ -6,12 +6,11 @@ export type Breakpoint = OverridableStringUnion<
   'xs' | 'sm' | 'md' | 'lg' | 'xl',
   BreakpointOverrides
 >;
-export type BreakpointValues = { [key in Breakpoint]: number };
 export const keys: Breakpoint[];
 
 export interface Breakpoints {
   keys: Breakpoint[];
-  values: BreakpointValues;
+  values: { [key in Breakpoint]: number };
   up: (key: Breakpoint | number) => string;
   down: (key: Breakpoint | number) => string;
   between: (start: Breakpoint | number, end: Breakpoint | number) => string;

--- a/packages/material-ui/src/styles/index.d.ts
+++ b/packages/material-ui/src/styles/index.d.ts
@@ -35,6 +35,8 @@ export {
   Direction,
   Breakpoint,
   BreakpointOverrides,
+  Breakpoints,
+  BreakpointsOptions,
   CreateMUIStyled,
   CSSObject,
 } from '@material-ui/system';

